server/tests/client.ts

```diff
@@ -47,8 +51,19 @@ describe('Test a client controllers', function () {
   before(async function () {
     this.timeout(120000)
 
-    server = await flushAndRunServer(1)
-    server.accessToken = await serverLogin(server)
+    servers = await flushAndRunMultipleServers(2)
```

Comment:
A fancy way would be `[server, secondServer] = await flushAndRunMultipleServers(2)`